### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by Ig0r17** [hannob/snallygaster#12]
Can you make a tutorial? Please

**Comment by spongyfrog** [comment]
yeah i think the author should give some few steps to use the app ... am getting some errors

**Comment by pkrahul87** [comment]
Yes. Just like others I think you should make a tutorial on how to use this app. I tried with all the possible cases but not getting any result.

**Comment by KommX** [comment]
Try `./snallygaster --help` or `./snallygaster -d github.com` to see how it works.

**Comment by Abiral1** [comment]
The tool only outputs when it finds something.
